sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -163,6 +164,7 @@ Use the `model` parameter to select the type of model for analysis.
 |-|-
 |`prebuilt-layout`| Text extraction, selection marks, tables
 |`prebuilt-document`| Text extraction, selection marks, tables, key-value pairs and entities
+|`prebuilt-read`|Text extraction and detected languages
```

Comment:
should we add prebuilt-tax.us.w2 here?

Reply:
I can't comment on the line, but on L428 we also have info on the prebuilts and their fields where we might want to include the W2 model as well.